client/src/app/core/services/pdf-document.service.ts

```diff
@@ -446,6 +459,21 @@ export class PdfDocumentService {
             },
             tocCategorySection: {
                 margin: [0, 0, 0, 10]
+            },
+            userDataTitle: {
+                fontSize: 26,
+                margin: [0, 0, 0, 0],
+                bold: true
+            },
+            tableHeader: {
+                bold: true,
+                fillColor: 'white'
+            },
+            tableEven: {
```

Comment:
We do not need `tableEven` and `tableOdd` anymore. PDFmake can do that now using table style functions.
I comment them bellow.